site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 2. The source code is available in a public git repository.
 3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
 4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
-5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```

Comment:
If the release_notes_url should point to the specific notes of the current version, it should be precised here.
(Looking at the [mattermost-server code](https://github.com/mattermost/mattermost-server/blob/57717a23afa2917e543d833ceb04634a26c01fd6/model/manifest.go#L114), the doc points to a specific version changelog, however, if a unique changelog page for all versions (such as the Github Releases section, or Mattermost own changelog page) is enough, there's no need to be more explicit in the doc.)

Reply:
Yes, `release_notes_url` should point to a specific notes.

Does this make it more clear:
```suggestion
5. For the current release and every upcoming ones a changelog has to be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```

Reply:
Yes, it is very clear, thank you.